node/api/channel_ws_api.md

```diff
@@ -1252,3 +1274,61 @@ The FSM will inform its client of each error, using a `conflict` report.
   "version": 1
 }
 ```
+
+#### Negative response
```

Comment:
Same here, how does that relate to signing error replies?

Reply:
Same as above: tthe signing error can have a negative reply